src/utils/colorManipulation.ts

```diff
@@ -13,20 +13,81 @@ export function computeBacklightFrame(
   videoDimensions: Dimensions,
   opts: BacklightOptions
 ) {
+  const newFrame = new Uint8ClampedArray(frame.data.length);
   const divisions = computeDivisions(
     frame,
     videoDimensions,
     opts.horizontalDivisions,
     opts.verticalDivisions,
     false
   );
+  const oneFourthDivWidth = Math.round(divisions[0].width / 4);
+  const oneFourthDivHeight = Math.round(divisions[0].height / 4);
 
-  divisions.forEach((division) => {
-    ctx.fillStyle = `rgb(${division.color[RED_CHANNEL_OFFSET]}, ${division.color[GREEN_CHANNEL_OFFSET]}, ${division.color[BLUE_CHANNEL_OFFSET]})`;
-    ctx.fillRect(division.col, division.row, division.width, division.height);
-  });
+  for(let div = 0; div < divisions.length; div += 2) {
```

Comment:
can you leave a comment here on why we're incrementing by 2 please? i believe it was to set two divisions at a time and blur them once they're set but that's not immediately clear